openmrs/apps/reports/RADET/sql/pmtct_hei.sql

```diff
@@ -1,42 +1,44 @@
-	0 as "Post Natal Health visit / Visite de santé post-natale",
-	0 as "Last ARV Pickup Date / Dernière date de prise en charge des ARV",
-	0 as "Days of ARV Refill / Jours de recharge ARV",
-	0 as "Location of ARV refill / Emplacement de la recharge d'ARV",
-	0 as "Differentiated ART delivery model at Last ARV refill ",
-	0 as "TB screening status at Last ARV Refill / Statut de dépistage de la tuberculose",
-	0 as "Current Viral Load/ Charge virale actuelle  (c/ml)",
-	0 as "Date of Current Viral Load / Date de la charge virale actuelle (dd-mmm-yyyy)",
-	0 as "Viral Load Indication/ Indication de la charge virale",
-	0 as "Maternal Outcome/ Résultat maternel",
-	0 as "Status of Missed appointment / Statut de rendez-vous manqué",
-	0 as "Child ID",
-	0 as "Date of birth (Date de naissance)",
-	0 as "Infant ARV Prophylaxis",
-	0 as "Date of Infant ARV Prophylaxis",
-	0 as "Infant CTX",
-	0 as "Date of Infant CTX",
-	0 as "Test type ",
-	0 as "Date of EID Sample Collected",
-	0 as "Date of EID Sample Sent",
-	0 as "Date EID Result Received at Facility ",
-	0 as "Date Caregiver Given EID Result",
-	0 as "EID Result ",
-	0 as "Month of Health Facility Visits when infant is seen at facility / Mois des visites dans un établissement de santé lorsqu'un nourrisson est vu dans un établissement",
-	0 as "Visit status / Statut de la visite",
-	0 as "Infant Outcome at 18 Months/ Résultat infantile à 18 mois",
-	0 as "Date of ART initiation (If HIV-positive) / Date d'initiation du ART (si HIV positif)",
-	0 as "Child ART Code",
-	0 as "Health Facility/  Établissement de santé",
-	0 as "Psychosocial Agents (Retention APS)";
\ No newline at end of file
+	"N/A" as "HTS Client ID",
+	getDateOfPositiveHIVResult(p.patient_id, "2010-01-01", "#endDate#") as "Date of HIV Positive / Date de séropositivité",
+	getPatientARTNumber(p.patient_id) as "Patient Unique ID/ART №/TARV № ",
+	getPatientARVStartDate(p.patient_id) as "ART Start Date/ Date de début de l'ART  (dd-MMM-yyyy)",
+	"N/A" as "Date of Delivery/ Date de livraison",
+	"N/A" as "Place of delivery / Lieu de livraison",
+	"N/A" as "Post Natal Health visit / Visite de santé post-natale",
+	getLastArvPickupDate(p.patient_id, "2010-01-01", "#endDate#") as "Last ARV Pickup Date / Dernière date de prise en charge des ARV",
+	getDurationMostRecentArvTreatment(p.patient_id, "2010-01-01", "#endDate#") as "Days of ARV Refill / Jours de recharge ARV",
+	getLocationOfArvRefill(p.patient_id, "2010-01-01", "#endDate#") as "Location of ARV refill / Emplacement de la recharge d'ARV",
+	"N/A" as "Differentiated ART delivery model at Last ARV refill ",
+	"N/A" as "TB screening status at Last ARV Refill / Statut de dépistage de la tuberculose",
+	getViralLoadTestResult(p.patient_id) as "Current Viral Load/ Charge virale actuelle  (c/ml)",
+	getViralLoadTestDate(p.patient_id) as "Date of Current Viral Load / Date de la charge virale actuelle (dd-mmm-yyyy)",
+	"N/A" as "Viral Load Indication/ Indication de la charge virale",
```

Comment:
@tmvumbi2 indication is the type of viral load test: Not documented, routine or targeted